ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -172,6 +174,17 @@ applyChainTick = lrResult ..: applyChainTickLedgerResult
 -------------------------------------------------------------------------------}
 
 -- | Ledger state associated with a block
+--
+-- This is the Consensus notion of a Ledger /ledger state/. Each block type is
```

Comment:
```suggestion
-- This is the Consensus notion of a /ledger state/. Each block type is
```

Reply:
This was intentional, because the Ledger's ledger state is not the ledger state of consensus. Removed in any case.